lightning/src/ln/onchaintx.rs

```diff
@@ -487,8 +487,8 @@ impl OnchainTxHandler {
 		let mut aggregated_soonest = ::std::u32::MAX;
 		let mut spendable_outputs = Vec::new();
 
-		// Try to aggregate outputs if they're 1) belong to same parent tx, 2) their
-		// timelock expiration isn't imminent (<= CLTV_SHARED_CLAIM_BUFFER).
+		// Try to aggregate outputs if their timelock expiration isn't imminent (absolute_timelock
+		// <= CLTV_SHARED_CLAIM_BUFFER) and they don't require an immediate CSV (aggregable).
```

Comment:
Nope, if we have to satisfy a CSV delay, we will mark them as spendable output, the non-aggregable one are timeout tx on remote commitment tx because we have to set nLocktime and that means we would have to pick up the highest nLocktime among the aggregated tx (explained in aggregable field)